### DISCUSSION TITLE
How to move the cursor back to the begging of the coding line

### DISCUSSION THREAD

**Discussion by silentwings2004** [cpeditor/cpeditor#1148] [Q&A]
Currently I found each time I want to move the cursor back to the beginning of the line where the code is, the cursor actually goes back to the head of the line where there is the space. Can anyone know how to do the above like most of the default setting of the other editor? Thanks!

for instance:

i want to move the cursor back to the place where | is:

      |while (true)

instead of the current setting as below:

| _______ while (true)

**Comment by ouuan** [comment]
This is not possible in the current version.

It's being tracked at #774. As a result of recent slow development and previous pull requests with unresolved problems, it has not been fixed for a long time.

I fixed it in https://github.com/cpeditor/QCodeEditor/pull/41 just now, and the fix will be in the next release.
